webapp/src/pages/Settings.tsx

```diff
@@ -783,6 +793,69 @@ const Settings: React.FC<Props> = ({ onClose }) => {
     </FormGroup>
   );
 
+  const getCommonFieldsConfigSection = () => (
+    <Box marginY={1.5}>
+      <FormGroup>
+        <Columns columns={4}>
+          {displayReadonlyFields(
+            "artifact_path",
+            resultingConfig.artifact_path
+          )}
+          <NumberField
+            key="batch_size"
+            label="batch_size"
+            value={resultingConfig.batch_size}
+            disabled={isUpdatingConfig}
+            onChange={(newValue) =>
+              setPartialConfig({
+                ...partialConfig,
+                batch_size: newValue,
+              })
+            }
+            {...FIELDS["batch_size"]}
+          />
+          <StringField
+            select
+            label="use_cuda"
+            className="fixedWidthInput"
+            value={String(resultingConfig.use_cuda)}
+            disabled={isUpdatingConfig}
+            onChange={(newValue: string | boolean) =>
+              newValue &&
+              setPartialConfig({
+                ...partialConfig,
+                use_cuda: newValue === "auto" ? newValue : Boolean(newValue),
+              })
+            }
+          >
+            {["auto", "true", "false"].map((item) => (
+              <MenuItem key={item} value={item}>
+                {item}
+              </MenuItem>
+            ))}
+          </StringField>
+          <FormControlLabel
+            control={
+              <Checkbox
+                size="small"
+                checked={Boolean(resultingConfig.large_dask_cluster)}
```

Comment:
`resultingConfig.large_dask_cluster` is already a boolean, no need to convert it
```suggestion
                checked={resultingConfig.large_dask_cluster}
```